WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -304,10 +287,10 @@ private extension PaymentCaptureOrchestrator {
 
         public var errorDescription: String? {
             switch self {
-            case .belowMinimumAmount(let amount):
-                return String.localizedStringWithFormat(Localizations.belowMinimumAmount, amount)
-            case .other:
-                return Localizations.defaultMessage
+                case .belowMinimumAmount(let amount):
+                    return String.localizedStringWithFormat(Localizations.belowMinimumAmount, amount)
+                case .other:
+                    return Localizations.defaultMessage
```

Comment:
super nit: similarly, it might be nice if we don't include these indentation changes above

Reply:
AppCode formats it this way, and apparently, there is not any kind of linter on this on CI. Ill revert formatting changes

Reply:
Reverted - https://github.com/woocommerce/woocommerce-ios/pull/6519/commits/7a94206c64ed11afad89a1c5a0e13e1222df068d